lib/Department.js

```diff
@@ -0,0 +1,67 @@
+const inquirer = require("inquirer");
+class Department {
+    constructor(conn) {
+        this.conn = conn;
+        this.getInfoQuery =  `select * from departments`;
+        this.addQuery = `insert into departments (name) values (?)`;
+        this.deleteQuery = `delete from departments where name = ?`;
+        this.addQuestions = [
+            {
+                type: "input",
+                name: "department_name",
+                message: "What is the name of the department?",
+            }
+        ],
+        this.deleteQuestions = [
+            {
+                type: "confirm",
+                name: "confirm_delete",
+                message: `Are you sure you want to delete department?`
+            }
+        ]
+        this.getBudgetsByDepartment = `select departments.name AS department_name, SUM(roles.salary) AS total_salaries
+        FROM departments JOIN roles ON roles.department_id = departments.id JOIN employees ON employees.role_id = roles.id
+        GROUP BY departments.name
+        `
+    }
+    getInfo(query) {
```

Comment:
if you would like to set a default value to the query param that you can do this here e.g.
```
getInfo(query = this.getInfoQuery) {}
```
If query is ever undefined or falsy, its value will be that of getInfoQuery :-)